js/iteratorAndGenerator.js

```diff
@@ -0,0 +1,38 @@
+// Task 1
+
+// let fizzBuzz = [];
+
+// fizzBuzz[Symbol.iterator] = function* () {
+//   for (let i = 1; i <= 100; i++) {
+//     if (i % 15 === 0) {
+//       yield "FizzBuzz";
+//     } else if (i % 5 === 0) {
+//       yield "Buzz";
+//     } else if (i % 3 === 0) {
+//       yield "Fizz";
+//     } else if (i === 100) {
+//       return "Buzz";
+//     } else {
+//       yield i;
+//     }
+//   }
+// };
+
+// for (item of fizzBuzz) {
+//   console.log(item);
+// }
+
+
+// Task 2
+
+function* generateRandomNumbers(max, quantity) {
+	for (let i=0; i < quantity; i++) {
+		yield Math.ceil(Math.random() * max)
+	}
+}
+
+const randomNumber = generateRandomNumbers(10, 4);
+
+for (value of randomNumber) {
```

Comment:
```suggestion
for (let value of randomNumber) {
```

Не забувайте про змінні та strict mode